sfdx-source/apex-common/main/classes/fflib_SObjectDomain.cls

```diff
@@ -298,6 +298,23 @@ public virtual with sharing class fflib_SObjectDomain
     {
     	return Records;
     }
+
+	/**
+	 * @param criteria The condition the returned record must meet
+	 *
+	 * @return Return the SObject records contained in the domain
+	 */
+	public virtual List<SObject> getRecords(fflib_Criteria criteria)
```

Comment:
If the domain layer and trigger handler PR is merged, then I will also move this into the fflib_SObjects class

Reply:
@wimvelzeboer   To which PR are you referring here?

However I think the one you're referring to was merged, and since this PR has conflicts, please take a look at those.

Reply:
Thanks @stohn777 completely forgot about this one, that I still had to rebase it after the merge of the new domain structure.

I think all looks good now. :-)